src/main/steps/applicant2Sequence.ts

```diff
@@ -184,6 +185,10 @@ const postSubmissionSequence: Step[] = [
     url: HUB_PAGE,
     getNextStep: () => HOME_URL,
   },
+  {
+    url: APP_REPRESENTED,
+    getNextStep: () => HOME_URL,
+  },
   {
     url: CONTINUE_WITH_YOUR_APPLICATION,
     getNextStep: data =>
```

Comment:
There are another set of application steps for the respondent ("respondentSequence") - do those need to be consistent?

Reply:
I'll check, ta, I'm not entirely sure this is essential but because the other pages have this HOME_URL as next step I'm adhering to what's there already